app.py

```diff
@@ -1,26 +1,24 @@
 from json import loads
+import logging
 
 import structlog
-from ras_common_utils.ras_config import ras_config
-from ras_common_utils.ras_logger.ras_logger import configure_logger
 from retrying import RetryError
 
+from logger_config import logger_initial_config
 from run import create_app, initialise_db
 
 """
 This is a duplicate of run.py, with minor modifications to support gunicorn execution.
 """
 
-logger = structlog.get_logger()
+logger = structlog.wrap_logger(logging.getLogger(__name__))
 
-
-config_path = 'config/config.yaml'
-config = ras_config.from_yaml_file(config_path)
```

Comment:
The 'from_yaml_file' also has logic which checks if there is a VCAP Env var which overrides the config file. this would break in the cloudfoundry environment because the database will not be configured properly.

This is the code from ras_common_utils:
 ```python
def from_yaml_file(path):
    with open(path) as f:
        data = yaml.load(f.read())
    return make(data)

def make(config_data):
    vcap_application = getenv('VCAP_APPLICATION')
    if vcap_application:
        log.info("CloudFoundry detected. Creating CloudFoundry configuration object.")
        return RasCloudFoundryConfig(config_data)
    else:
        log.info("CloudFoundry not detected. Creating standard configuration object.")
        return RasConfig(config_data)
```

Reply:
Looks like there's a lot of overcomplicated stuff going on to load from VCAP in common_utils.

This should be simple with the cfenv package. We're using it to [get redis config in frontstage](https://github.com/ONSdigital/ras-frontstage/blob/master/frontstage/cloud/cloudfoundry.py)